### PR TITLE
Laptop Price Predictor

### PR DESCRIPTION
**Pull request by Gaurav-576**
# Pull Request for DL-Simplified 💡

- **Info about the related issue (Aim of the project)** : A laptop price prediction model
- **Name:** Gaurav Kumar Singh
- **GitHub ID:** https://github.com/Gaurav-576
- **Email ID:** gauravsingh96753@gmail.com
- **Idenitfy yourself: GSSoC'24

Closes: #334 

### Describe the add-ons or changes you've made 📃
A laptop price prediction folder which consists of its respective dataset, models and images obtained during performing EDA on the dataset

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
1. Add a README file inside the Dataset folder and put a brief about the dataset.
2. Rename the `images` folder to `Images` folder.
3. Add a README file for this project, follow the [template](https://github.com/abhisheks008/DL-Simplified/blob/main/.github/readme_template.md). Document your project as per the template.

@Gaurav-576

File: Laptop Price Prediction Using DL/Model/README.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+**Univariate Analysis of the PPI(Pixels per Inches) with the Price of the Laptop**
+![graph](../Images/PPI_Price_Distribution.png)
+
+**Univariate Analysis of the Touchscreen property of the Laptop with the Price of the Laptop**
+![graph](../Images/Touchscreen_Laptops.png)
+
+**Univariate Analysis of the Weight of the Laptop with the Price of the Laptop**
+![graph](../Images/Laptop_Weights.png)
+
+**Correlation heatmap of the features**
```
Comment (abhisheks008) [inline Laptop Price Prediction Using DL/Model/README.md:130]: Please align this image properly.

**Review by abhisheks008 — CHANGES_REQUESTED** [review]

File: Laptop Price Prediction Using DL/Model/README.md (hunk 1 above)
Comment (Gaurav-576) [reply Laptop Price Prediction Using DL/Model/README.md:130]: Can I know what is the issue with this image? And what changes are you looking for?

Comment (abhisheks008) [reply Laptop Price Prediction Using DL/Model/README.md:130]: The heading and the image is showing in the same line, basically an indentation error in markdown. Hit an enter or use `</br>` tag for this.

Comment (Gaurav-576) [reply Laptop Price Prediction Using DL/Model/README.md:130]: Ok got it

**Review by abhisheks008 — APPROVED** [review]
Looks good to me! Your PR is approved and ready to be merged.

I'll merge this one by EOD.

@Gaurav-576

**Comment by Gaurav-576** [comment]
Great it was quite fun working on this project thanks a lot for this opportunity.

**Pull request merged**